R/utils.R

```diff
@@ -127,22 +126,24 @@ add_facet_labels <- function(p, xfacet_label = NULL, yfacet_label = NULL) {
 #' @param str_args (\code{character}) A character vector that the function shall
 #'  be executed with
 #'
-#' @return: call (i.e. expression) of the function provided by \code{fun}
+#' @return: Call (i.e. expression) of the function provided by \code{fun}
 #'  with arguments provided by \code{str_args}.
```

Comment:
Please remove the `\code`

Reply:
I already did it in https://github.com/insightsengineering/teal.modules.general/pull/680, get on with it!

Reply:
Ouch, Brace for some conflicts.
Did you cover all of the utils file?
Would you mind reviewing this PR @chlebowa (only if you have space for this)?